python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -126,10 +127,7 @@ def reload(self):
         self.registry._lock.acquire()
         try:
 
-            if self.registry.checkShouldFlush():
```

Comment:
It's being removed since the function `checkShouldFlush()` no longer exists. It was used previously to keep track of the dirtiness counter but now that it is time-based it is not used.